Makefile.PL

```diff
@@ -351,7 +351,7 @@ if (-d 'inc/patches') {
     if (which('patch')) {
         for my $file (glob 'inc/patches/*.patch') {
             # replacing \n with \n will change \n to \r\n on Win, without this patch.exe will crash
-            system "cd \Q$mxbase\E && perl -pe \"s/\\n/\\n/\" ../../\Q$file\E | patch -p1 -N";
+            system "cd $mxbase && perl -pe \"s/\\n/\\n/\" ../../$file | patch -p1 -N";
```

Comment:
There is no security issue here.
$mxbase is defined in line 335: my $mxbase = 'inc/matrixssl-3-7-2b-open';
and $file is one of the file in 'inc/patches'.

On my Windows setup I get for every patch attempt this:
"The system cannot find the path specified."